HelpSource/Classes/EnvGate.schelp

```diff
@@ -12,13 +12,13 @@ method::new
 Returns an link::Classes/EnvGen::.
 
 argument::i_level
-initial level of envelope (if set to 1, it starts open)
+initial level of envelope (if set to 1, it starts open). If nil, the initila level depends on the code::fadeTime:: control: if code::fadeTime <= 0::, it starts with 1.0 to let through impulses on the first sample. Otherwise it starts at 0.
```

Comment:
small spelling error : initila instead of initial

Reply:
thanks!